### DISCUSSION TITLE
issue?

### DISCUSSION THREAD

**Discussion by moojttaba** [react-hook-form/react-hook-form#7800] [Q&A]
![Screen Shot 2022-02-11 at 2 40 24 PM](https://user-images.githubusercontent.com/65502829/153581645-a9f5e829-9174-442c-9c5d-f6a2ab56047e.png)
`

**Comment by Moshyfawn** [comment]
This is **the most** unhelpful way of asking for support.
Please, consider creating a codesanbox reproduction with the issue you're facing.

Everything works as expected: [codesanbox](https://codesandbox.io/s/next-js-forked-i9gew?file=/pages/index.js)
